build/tools/ExtractRuleDescFromJson/RuleDescExtractor.cs

```diff
@@ -18,8 +18,8 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
```

Comment:
Silly question, I have used the word "xaml" often whereas you use "xml" is that something I should pay attention to when naming variables or?

Reply:
It depends on the content. `XML` is a general-purpose markup language that specifies how a document should be structured for it to be a "well-formed" XML document. It doesn't say anything about what the content of the document means.

`XAML` refers to a specific UI technology that happens to use XML as the serialization format. If we're talking about a XAML document, we expect the content to refer to XAML UI elements e.g. listboxes, labels, panels etc.

Here, we don't expect the content of the embedded files to be XAML, but we do expect they to be parseable as XML.